### DISCUSSION TITLE
Eclipse CDT development install requires JRE 11?

### DISCUSSION THREAD

**Discussion by tomoverlund** [eclipse-cdt/cdt#350] [Q&A] [answered]
I tried following the instructions here:

https://github.com/eclipse-cdt/cdt/blob/main/CONTRIBUTING.md

After dragging the link to the Oomph installer window, it wouldn't let me progress past page 3, "Variables", until filling in a value for "JRE 11 Location". I tried putting in a "." just to get past it, but it ends up with a stack trace later on:

```
Targlet container profile update completed
Performing API Baseline from Target Modular API Baseline, activate
Creating new baseline from target Modular API Baseline Target
Activating baseline: Modular API Baseline
Performing JRE for JavaSE-11 = .
Creating JRE for JavaSE-11 with location .
The location '.' is not valid
Go back to the Variables page, be sure to select 'Show all variables', and change the location to a valid one
ERROR: org.eclipse.jdt.launching code=0 Target is not a JDK Root. Java executable was not found
  at org.eclipse.oomph.setup.jdt.impl.JRETaskImpl$JREHelper.perform(JRETaskImpl.java:682)
```

Environment: WIndows 10. I installed Eclipse CDT yesterday with a fresh install via Oomph, which was also newly installed yesterday via the Eclipse.org site.

**Comment by ruspl-afed** [comment]
What exactly is unexpected here? Does it work for valid JRE 11 location?

**Reply by tomoverlund** [reply]
> What exactly is unexpected here? Does it work for valid JRE 11 location?

Why would I need to manually specify a JRE 11 location? Modern Eclipse comes with its own JRE. On the first page of options in the Oomph installer, there's an option of what JVM to use (17, 18, or 19).

**Reply by jonahgraham** [reply]
You can enter Java 17 in that box, which is probably why it wasn't noticed. Note that the JRE entered in that page does not need to match the JRE used to run Eclipse itself (the one from the first page).

**Comment by jonahgraham** [comment, marked as answer]
Thanks @tomoverlund for asking the question - the label is out of date and in #351 it is now fixed. Until the update is sync'd you can enter the Java 17 JRE in that location and it will work fine.
